src/transaction/atomic_replication_helper.cpp

```diff
@@ -365,6 +365,39 @@ namespace cublog
     return false;
   }
 
+#if !defined (NDEBUG)
+  void
+  atomic_replication_helper::atomic_log_sequence::dump ()
+  {
+    char buf[PATH_MAX];
```

Comment:
I'm worried that it's easy to overflow the buffer of PATH_MAX(256).

Reply:
Correct. I added some code to prevent overflow but not complete. I will complete it.